integration/noderunner/noderunner_test.go

```diff
@@ -83,8 +84,20 @@ func TestCanStartStandaloneObscuroHostAndEnclave(t *testing.T) {
 	network := gethnetwork.NewGethNetwork(int(gethPort), int(gethWebsocketPort), gethBinaryPath, 1, 1, []string{hostAddress.String()}, "", int(gethlog.LvlDebug))
 	defer network.StopNodes()
 
-	go enclaverunner.RunEnclave(enclaveConfig)
-	go hostrunner.RunHost(hostConfig)
+	enclCont := enclavecontainer.NewEnclaveContainer(enclaveConfig)
+	go func() {
+		err := enclCont.Start()
+		if err != nil {
+			panic(err)
+		}
+	}()
```

Comment:
Doing this on-the-fly, but I think it works.

```suggestion
	go func() {
		err := enclavecontainer.NewEnclaveContainer(enclaveConfig).Start()
		if err != nil {
			panic(err)
		}
	}()
```